references/kubernetes/cloud-resources/aws/dynamodb.yaml

```diff
@@ -4,34 +4,20 @@ apiVersion: dynamodb.services.k8s.aws/v1alpha1
 kind: Table
```

Comment:
It doesn't make sense to put this in the catalog as you need the ability to configure parameters